### DISCUSSION TITLE
Lists are useless. Lets use tags or keywords

### DISCUSSION THREAD

**Discussion by vredko** [community/community#85010] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi. 

Please let us use tags or keywords somewhere in depositories management routine. I can put them in description but its not what you expect us to do (maybe I'm wrong and you do expect that)

Lists are obsolete in (personal/scientific and etc) knowledge management without context, because they are hierarchical and eventually have dead ends and usage limitations. 

Tags\keywords, on the opposite side, are contextual. 

If some one interested in proofs and argumentations, I can and I will provide further evidence to my point with papers, researches and etc.

**With all respect and hope**

**Comment by artiphishle** [comment]
I also like to get this flexibility.

1 In the most easiest implementation it means you can add a repo to more than one list (and call the list names 'tags' then) right?

2 A more complex approach could be adding dimensions like creation date of a tag & individual tag coloring, to group some lists by another topic than the tagname.

I'm not sure if 2 would be too much (possibly), tbh I'm using lists here for a while now and I haven't missed tags (I expected to).

My listnames for now are distinguishable and I can easily find what I'm looking for.

I'm using lists are obvious enough to keep them in one list only,

However the flexibility tags would bring from user perspective would be very nice.

Adding a tag filter (omnibox) and some tags next to it, `often used` & `pinned` for quick selection would be useful I think.

Just saying, lists are useful already now I think.


More opinions? Kind regards, Fabio

**Reply by hzenooz** [reply]
- [ ]

**Comment by Al3676** [comment]
Thanks

**Reply by Al3676** [reply]
Good

**Comment by rasolAAaa** [comment]
- - [] بسیار زیبا
